### PR TITLE
[ruby/de] typo, formatting and translation fixes

### PR DESCRIPTION
**Pull request by denniskeller**

### CODE REVIEW

File: de-de/ruby-de.html.markdown [hunk 1]
```diff
@@ -11,6 +11,7 @@ contributors:
   - ["Dzianis Dashkevich", "https://github.com/dskecse"]
   - ["Levi Bostian", "https://github.com/levibostian"]
   - ["Rahil Momin", "https://github.com/iamrahil"]
+  - ["Dennis Keller", "https://github.com/denniskeller"]
```
Comment (Oire) [inline de-de/ruby-de.html.markdown]: Please move your name to `translators` section.

File: de-de/ruby-de.html.markdown [hunk 2]
```diff
@@ -232,7 +231,7 @@ hash.keys #=> ['color', 'number']
 hash['color'] #=> 'green'
 hash['number'] #=> 5
 hash['nothing here'] #=> nil 
-// Asking a hash for a key that doesn't exist returns nil:
+// Fragen für einen Hash, der nicht existiert liefert nil:
```
Comment (Oire) [inline de-de/ruby-de.html.markdown]: Fragen den Hash für einen Schlüssel, der nicht existiert?

**Comment by vendethiel** [comment]
@denniskeller any news? :)

**Comment by denniskeller** [comment]
Thanks for your Feedback! Is everything alright now? :)

**Comment by vendethiel** [comment]
@Oire ?

**Comment by vendethiel** [comment]
Going to take this one for good. Thanks!

**Pull request merged**